src/utils/__tests__/id_generator.test.ts

```diff
@@ -16,7 +16,7 @@
 
 /* eslint-disable @typescript-eslint/no-unsafe-member-access */
 
-import globalScope from "../../compat/global_scope";
+import globalScope from "..//global_scope";
```

Comment:
this import seems wrong

Reply:
:O I guess it weirdly works but you're right